app/config/logging-cfg-local.yml

```diff
@@ -68,7 +69,7 @@ formatters:
 handlers:
   console:
     class: logging.StreamHandler
-    formatter: standard
+    formatter: json
```

Comment:
Please keep the local to use standard, the other two configurations uses json format.

Reply:
reverted